finance/proposal-calls/cycle3/user-dev-support-rse.md

```diff
@@ -0,0 +1,86 @@
+### Title
+
+Research Software Engineer: User and Developer Support
+
+### Project Team
+
+- Moritz Guenther
+- Adrian Price-Whelan (original plan)
+
+
+### Project Description
+
+A research software engineer (RSE) position  \to
+tackle both user-facing and developer-facing support issues, as identified and
+prioritized by the Astropy community and infrastructure teams. The RSE would have a
+"manager" who is an existing Astropy contributor with existing knowledge of the Astropy
+core package, ecosystem, infrastructure packages, and community. The RSE is expected to
+eventually work independently, but with feedback from the Manager and Astropy community.
+We envision about an equivalent ~50% FTE appointment, but that can be scaled up or down
+based on the available budget.
+
+### Project / Work
+
+The Astropy Project is seeking to hire a full-time or part-time research software engineer to develop and implement new features that the community has identified as high priorities in the roadmap but that the volunteer contributors do not have the capacity to deliver, as well as address smaller, but long-standing bugs and issues in the astropy core package and the coordinated packages.
```

Comment:
> The Google doc says 50%, so that is part time only.

The google doc is not authoritative either. In an ideal world, I would offer 50% for 12 months or 100 % for 6 months or something in between at the candidates discretion. However, that depends on the time scale on which the Moore funding needs to be spend out.

Reply:
In that case, we need to specify that this is a termed position, which is unclear by stating "full-time" on top of the Google doc without further context. Do you agree?

Reply:
I agree - but I'm waiting for a final budget from the finance committee to make all the percentage of effort / pay per hour / length of appointment consistent.

Reply:
@hamogu - Finance committee just discussed this and here's what we settled on: The RSE budget is up to 75k in consulting fees, to be spent by March 31, 2024, the end date of Moore grant. If a 12-month contract is needed by a particularly RSE, funds could be allocated from NASA. Is that sufficient for you to carry forward?

Reply:
Yes, I will finalize this text and the job ad I've drafted.

Reply:
I still see "full-time or part-time" in the text. Shouldn't it now be "full-time for 6 months"?

Reply:
No. I think the developer has the option to work either full-time for 6 months or part time for 9 month or anything in between.

Reply:
But you are funded for 6 months only, how does the manager thingy works if this person working in a time frame when you cannot?

Reply:
I am funded for 0.1 FTE and I will spread that FTE over the same period that the RSE is active. If the RSE is a 6 months contract, then I'll to 1 day / week = 20% for 6 months and 0 % for the following 6 months, which works out to be 0.1FTE for an annual sub-grant. If the RSE is on for 12 month, then I'll do 0.5 days/week = 10 % for 12 months, and for anything in between, we can do that math.

In practice, I expect that to be uneven anyway, where I will spend considerably more time in the beginning and less later. And that's OK, there is no provision that I bill the exact same number of hours every week.

Reply:
Should this expectation be clarified in the formal proposal?

Reply:
It does not say "evenly distributed". Yes, I can tweak is even more, but I'd rather spend the time working with the RSE then to spend any more effort fine-tune this text any more. I think we all understand how I intend this to happen and if I don't do a good job, the Coco and the finance committee always have the option to terminate the contract and fire me.

Reply:
I just want things to be clear enough that if someone new (in whatever role) takes over and has to understand what is going on here, they can find out without having to ask around. 😸

Reply:
OK, I'll try to expand that section in the text.